### PR TITLE
Add release note for the informer cache memory consumption

### PR DESCRIPTION
**Pull request by ywk253100**
Add release note for the informer cache memory consumption

Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by Lyndon-Li — DISMISSED** [review]

**Review by blackpiglet — DISMISSED** [review]

File: changelogs/CHANGELOG-1.13.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ To fix CVEs and keep pace with Golang, Velero made changes as follows:
 ### Limitations/Known issues
 * The backup's VolumeInfo metadata doesn't have the information updated in the async operations. This function could be supported in v1.14 release. 
 
+### Note
+* Velero may use more memory due to the introduction of informer-cache, increase the memory limit if restoring large scale of resources
```
Comment (anshulahuja98) [inline changelogs/CHANGELOG-1.13.md:68]: can we make it clear based on the parameter of disableInformerCache

SOmething like if disableInformerCache = false => equivalent to existing velero behaviour
= true => better restore performance but higher memory usage.

**Review by anshulahuja98 — CHANGES_REQUESTED** [review]

File: changelogs/CHANGELOG-1.13.md (hunk 1 above)
Comment (ywk253100) [reply changelogs/CHANGELOG-1.13.md:68]: @anshulahuja98 How about this one?

Velero introduces the informer cache which is enabled by default. The informer cache improves the restore performance but may cause higher memory consumption. Increase the memory limit of the Velero pod or disable the informer cache by specifying the `--disable-informer-cache` option when installing Velero if you get the OOM error.

Comment (anshulahuja98) [reply changelogs/CHANGELOG-1.13.md:68]: Seems good to me. Thank you for taking this up.

Comment (anshulahuja98) [reply changelogs/CHANGELOG-1.13.md:68]: CC: @sseago as FYI

**Review by anshulahuja98 — APPROVED** [review]

**Review by blackpiglet — APPROVED** [review]

**Pull request merged**